ignite/handlers/checkpoint.py

```diff
@@ -165,13 +168,12 @@ class Checkpoint(Serializable):
 
     Note:
         This class is distributed configuration-friendly: it is not required to instantiate the class in rank 0 only
-        process. This class supports automatically distributed configuration and if used with
-        :class:`~ignite.handlers.DiskSaver`, checkpoint is stored by rank 0 process.
+        process.
 
     .. warning::
 
-        When running on XLA devices, it should be run in all processes, otherwise application can get stuck on
-        saving the checkpoint.
+        When running on XLA devices or using :class:`~torch.distributed.optim.ZeroRedundancyOptimizer`, it
+        should be run in all processes, otherwise application can get stuck on saving the checkpoint.
```

Comment:
We may need to rephrase this sentence...

Reply:
How exactly do you mean?